libsolidity/formal/SMTEncoder.cpp

```diff
@@ -1524,40 +1495,32 @@ void SMTEncoder::arithmeticOperation(BinaryOperation const& _op)
 {
 	auto type = _op.annotation().commonType;
 	solAssert(type, "");
-	if (type->category() == Type::Category::Integer || type->category() == Type::Category::FixedPoint)
+	solAssert(type->category() == Type::Category::Integer || type->category() == Type::Category::FixedPoint, "");
+	switch (_op.getOperator())
 	{
-		switch (_op.getOperator())
-		{
-		case Token::Add:
-		case Token::Sub:
-		case Token::Mul:
-		case Token::Div:
-		case Token::Mod:
-		{
-			auto values = arithmeticOperation(
-				_op.getOperator(),
-				expr(_op.leftExpression()),
-				expr(_op.rightExpression()),
-				_op.annotation().commonType,
-				_op
-			);
-			defineExpr(_op, values.first);
-			break;
-		}
-		default:
-			m_errorReporter.warning(
-				5188_error,
-				_op.location(),
-				"Assertion checker does not yet implement this operator."
-			);
-		}
+	case Token::Add:
+	case Token::Sub:
+	case Token::Mul:
+	case Token::Div:
+	case Token::Mod:
+	{
+		auto values = arithmeticOperation(
+			_op.getOperator(),
+			expr(_op.leftExpression()),
+			expr(_op.rightExpression()),
+			_op.annotation().commonType,
+			_op
+		);
+		defineExpr(_op, values.first);
+		break;
 	}
-	else
+	default:
 		m_errorReporter.warning(
-			9011_error,
+			5188_error,
```

Comment:
No need to change the number imho, but not too important anyhow.

Reply:
I didn't really add new numbers, I suspect there were two errors in this function, I removed one and the GitHub diff thought the second replaced the removed one